exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,6 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var Total = numberOfStudents + numberOfMentors;
```

Comment:
by convention, variables in JavaScript are `lowerCamelCase`; variables that start with capital letters usually refer to a `class` (a data structure that you will learn about soon).